File: src/JavaScript/tasks/CombineTwoWords.js


const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Please enter the first string: ", (userInput1) => {
    rl.question("Please enter the second string: ", (userInput2) => {
        let combine = userInput1 + userInput2;


if (userInput1.charAt(userInput1.length - 1) === userInput2.charAt(0)) {

    combine = userInput1 + userInput2.substring(1);
}

        console.log(combine);
        rl.close();
    });
});

File: src/Java/tasks/CombineTwoWords.java
package Java.tasks;

import java.util.Scanner;

public class CombineTwoWords {

    public static void main(String[] args) {
        Scanner input =  new Scanner(System.in);
        System.out.println("Enter 2 string : ");
        String str1 = input.next();
        String str2 = input.next();
        String combine = str1 + str2;

        if(str1.charAt(str1.length()-1) == str2.charAt(0)){
            combine = str1 + str2.substring(1);


        }

        System.out.println(combine);
    }

}
